docs/timelion/getting-started/timelion-conditional.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[timelion-conditional]]
+=== Using conditional logic and tracking trends
+
+In this section you will learn how to modify a series with conditional logic and trend your data with a moving average. This is helpful to easily distinguish outliers and detect trends over time.
+
+For the purposes of this tutorial, you will continue to use https://www.elastic.co/downloads/beats/metricbeat[Metricbeat data] to add another visualization that monitors memory consumption. To start, use the following expression to chart the maximum value of  `system.memory.actual.used.bytes`.
+
+[source,text]
+----------------------------------
+.es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes')
+----------------------------------
+
+image::images/timelion-conditional01.png[]
+{nbsp}
+
+Let’s create two thresholds to keep an eye on the amount of memory your machine is using. For the purposes of this tutorial, your warning threshold will be 12.5GB and your severe threshold will be 15GB. When the maximum amount of memory being utilized exceeds either of these thresholds, the series will be re-styled accordingly.
+
+NOTE: If the threshold values are too high or low for your machine, please adjust accordingly.
+
+To configure these two threshold values, you can utilize Timelions conditional logic. In this tutorial you will use `if()` to compare each point to a number, adjust the styling if the comparison is `true` and use the default styling if the comparison is `false`. Timelion offers six operator values for comparison.
```

Comment:
if the comparison is `true` and use the default styling if the comparison is `false` => if the condition evaluates to `true` and use the default styling if the condition evaluates to`false`